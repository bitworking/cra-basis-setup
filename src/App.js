import React, { Component } from 'react';
import Headline from './components/atoms/Headline/Headline';

import TileMap from './components/molecules/TileMap/TileMap';

class App extends Component {
  render() {
    return (
      <div>
        <div>
          <Headline>Hexagon Strategy Game</Headline>
        </div>
        <TileMap />
      </div>
    );
  }
}

export default App;
