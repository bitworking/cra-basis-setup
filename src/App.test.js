import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

it('renderd app snapshot', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
