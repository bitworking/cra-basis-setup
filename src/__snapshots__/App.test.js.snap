// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderd app snapshot 1`] = `
<div>
  <div>
    <h1
      className="atom_headline-0-1 atom_headline-0-0"
    >
      Welcome to Reasdfsdfsdfsct
    </h1>
  </div>
  <p>
    To get started, edit 
    <code>
      src/App.js
    </code>
     and save to reload.
  </p>
</div>
`;
